src/components/Menu/Menu.module.css

```diff
@@ -18,7 +20,12 @@
 }
 
 .menu__button.menu__button {
+  @mixin eds-theme-typography-body-text-md;
```

Comment:
some of the styles being overridden are b/c `ClickableStyle` has a few things that conflict here

Reply:
Q for my learning: what does @mixin do here? A quick google mentions sass mixins but I don't think that's what this is.

Reply:
I'll ask @dierat / @jinlee93 to add context as I'm actually not sure how this is made to work using CSS Modules 🤔

If it works like Sass would, it basically imports a snippet of css properties and values into the current rule. I think if you search for `@define-mixin` you can see what properties it will grab

Reply:
Yea, this specific mixin is defined in [`typography-usage.css`](https://github.com/chanzuckerberg/edu-design-system/blob/next/src/design-tokens/tier-2-usage/typography-usage.css). 

It is sass like lang provided via [postcss](https://github.com/postcss/postcss-mixins) for bundling styling together. Being said, we can probably start using the [Tailwind @layer directive](https://tailwindcss.com/docs/functions-and-directives#layer) instead so we don't have confusion like this and reduce our dependency count.

I can investigate that